src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java

```diff
@@ -47,41 +51,59 @@ public void handle(BadSqlGrammarException ex) {
             "{} insert into error! - {}",
             this.getClass().getSimpleName(),
             ex.getR2dbcException().getMessage());
-        throw new CreateException("Trying to insert a string value that is too long");
+        throw ConcreteRequestErrorMessageException.invalidParameter("Trying to insert a string value that is too long");
       } else {
         log.debug(
             "{} update error! - {}",
             this.getClass().getSimpleName(),
             ex.getR2dbcException().getMessage());
-        throw new UpdateException("Trying to update a string value that is too long");
+        throw ConcreteRequestErrorMessageException.invalidParameter("Trying to update a string value that is too long");
       }
     } else if ("42804".equals(ex.getR2dbcException().getSqlState())) {
-      throw new DatabaseException(
+      throw ConcreteRequestErrorMessageException.internalServerError(
           "Internal mismatch between backEnd and database - please see log", ex);
     } else {
-      throw new DatabaseException("Internal error with database operation - please see log", ex);
+      throw ConcreteRequestErrorMessageException.internalServerError(
+              "Internal error with database operation - please see log", ex);
     }
   }
 
   @ExceptionHandler(ServerWebInputException.class)
   public void handle(ServerWebInputException ex) {
     if (ex.getMessage() != null && ex.getMessage().contains("Invalid UUID string:")) {
-      throw new InvalidParameterException("Input was not a valid UUID format");
+      throw ConcreteRequestErrorMessageException.invalidParameter("Input was not a valid UUID format", ex);
     } else {
       throw ex;
     }
   }
 
+  @ExceptionHandler(ConcreteRequestErrorMessageException.class)
```

Comment:
I'd suggest to have this handler at the very top above the on for DCSAException (as I think the order of handler matters kinda like a try-catch with multiple exceptions, but not entirely certain).